tsconfig.test.json

```diff
@@ -4,5 +4,5 @@
     "noEmit": true
   },
   "include": ["**/*.test.ts"],
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules", "apps/docs-snippets2/src/**/*.test.ts"]
```

Comment:
Why was this necessary?

Reply:
Strangely, this was the only way I could get the linter to ignore top-level _await_ despite us [enabling ES2022](https://github.com/FuelLabs/fuels-ts/blob/mc/continue-infra-for-snippets/apps/docs-snippets2/tsconfig.json#L4) for this package, not sure why though.

Reply:
Hmm it looks like some misconfig.

The problem with ignoring these files is that it'll silence everything.

Reply:
This file is responsible for compiling the test files and so it overrides any subdirectory _tsconfigs_ , including the one local to `docs-snippets2` 

But given we already run the compiler against the snippet files themselves, we shouldn't really need to run the typescript compiler against the generated test files, since they aren't tracked in the repo and any potential issues would result in test failures regardless.